docker_squash/lib/common.py

```diff
@@ -31,10 +31,17 @@ def docker_client(log):
     except KeyError:
         pass
 
-    params = docker.utils.kwargs_from_env()
+    params = {"version": "auto"}
+    params.update(docker.utils.kwargs_from_env())
     params["timeout"] = timeout
+
+    try:
+        APIClientClass = docker.Client  # 1.x
+    except AttributeError:
+        APIClientClass = docker.APIClient  # 2.x
+
```

Comment:
cant it be done in import section like 
try:
     from docker import client as APIClientClass
except:
     from docker import ApiClient as APIClientClass

or if its need to be there can you do probably docker.version check instead of try/except

I would expect it there probably